include/exiv2/easyaccess.hpp

```diff
@@ -19,73 +27,110 @@ namespace Exiv2 {
 // class declarations
 class ExifData;
 
-//! Return the orientation of the image
+//! Return the orientation of the image. Please keep in mind that this accessor is provided for convenience only and
+//! will return expected/desired metadata most of the time.
 EXIV2API ExifData::const_iterator orientation(const ExifData& ed);
-//! Return the ISO speed used to shoot the image
+//! Return the ISO speed used to shoot the image. Please keep in mind that this accessor is provided for convenience
+//! only and will return expected/desired metadata most of the time.
 EXIV2API ExifData::const_iterator isoSpeed(const ExifData& ed);
```

Comment:
As there is some extra computation involved for ISO, maybe another sentence (similar to lens detection) is warranted?